src/main/java/assignment2/utils/InputValidators/CoursesValidator.java

```diff
@@ -0,0 +1,27 @@
+package assignment2.utils.InputValidators;
+
+import assignment2.exceptions.InvalidException;
+import assignment2.exceptions.InvalidInputException;
+import assignment2.utils.CourseOption;
+import java.util.HashSet;
+
+public class CoursesValidator {
+
+  public static void isCoursesValid(String courses) throws InvalidException {
```

Comment:
move course options to enums and validate enum values directly.

Reply:
done.

Reply:
done.